buildscripts/setup.gradle

```diff
@@ -39,6 +43,80 @@ task renameAppPackage(type: Copy) {
     }
 }
 
+task renameMviPackage(type: Copy) {
```

Comment:
This is very similar in flow to `renameAppPackage`. If we continue adding more optional modules, we will want to look at a way to have a single `renameModule` task, that can take in some parameter for the relevant directory and clean itself up.